src/demo/app/components/directive-button-demo.component.ts

```diff
@@ -1,18 +1,26 @@
-import { Component } from '@angular/core';
```

Comment:
please revert all changes to this file.  This is not something that should have been included in this pull request.

Reply:
Going to comment this on the timeline, we spoke about it.